action.js

```diff
@@ -275,24 +275,33 @@ async function findTaskInSection(client, sectionId, name) {
             } else {
                 console.info('task found', task.gid);
                 existingTaskId = task.gid
-            }            
+            }
         });
     } catch (error) {
         console.error('rejecting promise', error);
     }
     return existingTaskId
 }
 
-async function createTask(client, name, description, projectId, sectionId = '', tags = [], collaborators = []) {
+async function createTask(client, name, description, projectId, sectionId = '', tags = [], collaborators = [], assignee = '', category = '') {
     const taskOpts = {
-        name: name, 
-        notes: description, 
+        name: name,
+        notes: description,
         projects: [projectId],
         tags,
         followers: collaborators,
         pretty: true
     };
 
+    if (assignee != '') {
+        taskOpts.assignee = assignee;
+    }
+
+    if (category === 'Pixels') {
```

Comment:
Not sure about this... This function is meant to be generic - we don't want to start inserting project-specific items here. Perhaps a separate action/function that takes in task ID, and the custom field IDs to set? Would be nice to keep it here, but GHA is limited in what type of objects can be passed in (i.e. I don't think we can easily pass in a map).

BTW, I found an [easy way](https://developers.asana.com/reference/getproject) to find these IDs for projects. Example for triage: https://app.asana.com/api/1.0/projects/69071770703008

Reply:
We could pass in a hash via a JSON encoded string.  Not pretty but should be safe enough?

Reply:
Played around with it a bit and looks like passing JSON string works! Let's do that :)